source/reading.md

```diff
@@ -607,195 +592,186 @@ using Python with SQLite and PostgreSQL databases.
 SQLite is probably the simplest relational database system
 that one can use in combination with Python. SQLite databases are self-contained and
 usually stored and accessed locally on one computer. Data is usually stored in
-a file with a `.db` extension. Similar to Excel files, these are not plain text
-files and cannot be read in a plain text editor. 
+a file with a `.db` extension (or sometimes a `.sqlite` extension). 
+Similar to Excel files, these are not plain text files and cannot be read in a plain text editor. 
 
-```{index} database; connect, SQLAlchemy, SQLAlchemy; create_engine, database; SQLAlchemy
+```{index} database; connect, ibis, ibis; ibis
 ```
 
-```{index} see: SQLAlchemy; database
+```{index} see: ibis; database
 ```
 
 The first thing you need to do to read data into Python from a database is to
-connect to the database. We do that using the `create_engine` function from the
-`sal` (SQLAlchemy) package. This does not read
+connect to the database. For an SQLite database, we will do that using 
+the `connect` function from the
+`sqlite` backend in the
+`ibis` package. This command does not read
 in the data, but simply tells Python where the database is and opens up a
 communication channel that Python can use to send SQL commands to the database.
 
+> **Note:** There is another database package in python called `sqlalchemy`. 
+> That package is a bit more mature and popular than `ibis`,
+> so if you want to dig deeper into working with databases in Python, that is a good next 
+> package to learn about. We will work with `ibis` in this book, as it 
+> provides a friendlier syntax that is more like `pandas` for data analysis code. 
```

Comment:
```suggestion
> provides a modern and friendlier syntax that is more like `pandas` for data analysis code. 
```

Just so that it doesn't sound like we are using some unpopular and not-preferred package